### PR TITLE
fix: [IXSPD1-1906] fix calculation of airdrop token amount

### PR DESCRIPTION
**Pull request by galvin96**
## Description

Please describe the purpose of this pull request.

## Changes

- add big.js
- calculate token amount from csv file with big.js

## Attached Links

1. Jira ticket: [IXSPD1-1906]
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

[IXSPD1-1906]: https://investax.atlassian.net/browse/IXSPD1-1906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Uploader/index.tsx [hunk 1]
```diff
@@ -122,14 +123,14 @@ export const Uploader: FC<UploaderProps> = ({
       setCsvRows([])
       onCsvRowsChange?.([])
       setTotalWallets?.(0)
-      setTotalAmount?.(0)
+      setTotalAmount?.('0')
     } else {
       setCsvRows(validatedRows)
       onCsvRowsChange?.(validatedRows)
       const totalWallets = validatedRows.length
-      const totalAmount = validatedRows.reduce((sum, row) => sum + parseFloat(row[1]), 0)
```
Comment (thi-investax) [inline src/components/Uploader/index.tsx:130]: hi @galvin96 , I'm curious why parseFloat fail for the previous case? Do you have any ideas?

**Comment by thi-investax** [comment]
@galvin96 ga

can you help update this line       showError('Invalid file: The file headers must be "WalletAddress" and "amount".')

src/components/Uploader/index.tsx


'Invalid file: The file headers must be "walletAddress" and "amount"

**Comment by galvin96** [comment]
Updated. @thi-investax

**Comment by thi-investax** [comment]
thanks @galvin96

**Pull request merged**